setup.py

```diff
@@ -35,7 +34,7 @@ def find_version(*file_paths):
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 test_deps = [
     'mock;python_version<"3.3"',
-    'pytest==4.5',
+    'pytest>=3.1',
```

Comment:
pytest 4.6.x breaks coverage

Reply:
hopefully not anymore ;) works here locally

Reply:
yea, i'll revert that for now